sycl/include/CL/sycl/event.hpp

```diff
@@ -24,6 +24,10 @@ namespace detail {
 class event_impl;
 }
 
+/// An event objects an be used to synchronize memory transfers, enqueues of
```

Comment:
**An event objects an be used to synchronize** memory transfers, enqueues of

I feel like some English grammar rules are broken here.